pkg/project/testoutput/images.yaml

```diff
@@ -0,0 +1,3 @@
+github.com/openshift-knative/hack/cmd/generate: hello
```

Comment:
Not sure I fully understand this file, why/what

Reply:
Added comments on the code that generates it.